README.md

```diff
@@ -39,6 +39,8 @@ To get started, install the packages using npm.
 
 `npm install`
 
+Copy the `.env.production` file to `.env.local` and fill in the missing values.
```

Comment:
I suppose this might be incorrect. Maybe we could create some `.env.example` template instead?

Reply:
I'm introducing one for the backend here https://github.com/stiftelsen-effekt/effekt-backend/pull/506

Reply:
Added .env.example